### PR TITLE
[zh-cn]: Update translation of String.CaseConvert()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#13975

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/string/tolocalelowercase/index.md [hunk 1]
```diff
@@ -5,46 +5,45 @@ slug: Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase
...
 
-根据任何特定于语言环境的案例映射规则将被调用字串转换成小写格式的一个新字符串。
+    与使用 `locales` 参数的其他方法不同，`toLocaleLowerCase()` 不允许进行区域设置匹配。因此，在检查 `locales` 参数的有效性之后，`toLocaleLowerCase()` 始终使用列表中的第一个区域设置（如果列表为空，则使用默认区域设置），即使该区域设置在实现中不受支持。
 
-### Exceptions
+### 返回值
 
-- A {{jsxref("RangeError")}} ("invalid language tag: xx_yy") is thrown if a `locale` argument isn't a valid language tag.
-- A {{jsxref("TypeError")}} ("invalid element in locales argument") is thrown if an array element isn't of type string.
+一个新的字符串，表示调用字符串根据特定区域设置的大小写映射规则转换为小写形式。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/tolocalelowercase/index.md:29]: verb to adj.

```suggestion
一个新的字符串，表示调用字符串根据特定区域设置的大小写映射规则转换得到的小写形式。
```

File: files/zh-cn/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md [hunk 2]
```diff
@@ -5,42 +5,39 @@ slug: Web/JavaScript/Reference/Global_Objects/String/toLocaleUpperCase
...
 
-A new string representing the calling string converted to upper case, according to any locale-specific case mappings.
+    与使用 `locales` 参数的其他方法不同，`toLocaleUpperCase()` 不允许进行区域设置匹配。因此，在检查 `locales` 参数的有效性之后，`toLocaleUpperCase()` 始终使用列表中的第一个区域设置（如果列表为空，则使用默认区域设置），即使该区域设置在实现中不受支持。
 
-### Exceptions
+### 返回值
 
-- A {{jsxref("RangeError")}} ("invalid language tag: xx_yy") is thrown if a `locale` argument isn't a valid language tag.
-- A {{jsxref("TypeError")}} ("invalid element in locales argument") is thrown if an array element isn't of type string.
+一个新的字符串，表示调用字符串根据特定区域设置的大小写映射规则转换为大写形式。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md:29]: same

**Review by yin1999 — COMMENTED** [review]
Great @JasonLamv-t. But could we improve these two translations.

File: files/zh-cn/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md (hunk 2 above)
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md:29]:
```suggestion
一个新的字符串，表示调用字符串根据特定区域设置的大小写映射规则转换得到的大写形式。
```

**Review by yin1999 — APPROVED** [review]
Thanks @JasonLamv-t

**Pull request merged**